app/views/WorkspaceView/index.tsx

```diff
@@ -12,23 +12,26 @@ import { getShowLoginButton } from '../../selectors/login';
 import ServerAvatar from './ServerAvatar';
 import styles from './styles';
 
-class WorkspaceView extends React.Component {
-	static navigationOptions = () => ({
-		title: I18n.t('Your_workspace')
-	});
+interface IWorkSpaceProp {
+	navigation: StackNavigationProp<any, 'WorkspaceView'>; // waiting for the RootStackParamList https://reactnavigation.org/docs/typescript/#type-checking-screens
```

Comment:
Can you add a // TODO here please ? 
like `// TODO - // waiting for the RootStackParamList ...`

Reply:
For sure!